assets/javascripts/modules/ajaxFormSubmit.js

```diff
@@ -55,6 +55,13 @@ var ajaxFormSubmit = {
         $form.on('keypress', 'input[type="text"], textarea', eventData, _this.keypressHandler);
       }
     }
+
+    $(document.body).on('registerNewForm', function (event, form) {
```

Comment:
I would replace this with a delegate event on the parent container that has the forms ajaxed into it